core/src/test/scala/integration/kafka/zk/ZkMigrationIntegrationTest.scala

```diff
@@ -73,7 +73,7 @@ object ZkMigrationIntegrationTest {
       MetadataVersion.IBP_3_7_IV2,
       MetadataVersion.IBP_3_7_IV4,
       MetadataVersion.IBP_3_8_IV0,
-      MetadataVersion.IBP_4_0_IV0
+      MetadataVersion.IBP_3_9_IV0
```

Comment:
(1) I guess we don't support ZK migration from 4.0. Could we add a comment?
(2) We have the following code in this class. Should we bump IBP_3_8_IV0 to IBP_3_9_IV0?

`@ClusterTest(types = Array(Type.ZK), brokers = 3, metadataVersion = MetadataVersion.IBP_3_8_IV0, serverProperties = Array(
`

Reply:
1. added

2. changed